example/90-shoot.yaml

```diff
@@ -350,6 +350,10 @@ spec:
 #   coreDNS:
 #     autoscaling:
 #       mode: horizontal # {horizontal,cluster-proportional}
+#     rewriting:
```

Comment:
I think this deserves some user-facing documentation under `docs/usage`.
Please explain what it exactly does, what the caveats are, etc.

Reply:
Good point. I will check and most likely adapt the `CoreDNS` documentation.

Reply:
Added documentation with new commit.